src/lean_spec/subspecs/containers/block/block.py

```diff
@@ -177,24 +181,25 @@ def verify_signatures(
         ):
             validator_ids = aggregated_attestation.aggregation_bits.to_validator_indices()
 
-            assert len(aggregated_signature) == len(validator_ids), (
-                "Aggregated attestation signature count mismatch"
-            )
-
-            attestation_root = aggregated_attestation.data.data_root_bytes()
+            attestation_data_root = aggregated_attestation.data.data_root_bytes()
```

Comment:
Quite minor and I should've raised in the previous PR. But just want to flag this for quick opinion @tcoratger spec-writing wise...

Since `data_root_bytes()` is doing just `bytes(hash_tree_root(self))`, should we do this instead and remove `data_root_bytes()`?

```suggestion
            attestation_data_root = hash_tree_root(aggregated_attestation.data)
```

also the `bytes(...)` can be handled inside `verify_aggregated_payload()` so we don't lose type information early

Reply:
I'm open to both; I don't have a preference. It seems to me that this pattern is used a lot in the code, so perhaps having the utility is simpler and less verbose.

Of course, if there are only one or two uses across the entire codebase, that is useless from a spec point of view.